aten/src/ATen/native/quantized/cpu/qconv_prepack.cpp

```diff
@@ -211,17 +211,30 @@ class QConvPackWeightInt8 final : public c10::OperatorKernel {
 
     auto weight_contig = weight.contiguous(MemoryFormat::ChannelsLast);
     auto bias_contig = bias.contiguous();
+    auto weight_zp = weight.q_zero_point() + 128;
+
+    int8_t* w_data = (int8_t*)weight_contig.data_ptr<c10::qint8>();
+    Tensor qnnp_weight = at::_empty_affine_quantized(
+        weight_contig.sizes(),
+        at::device(kCPU).dtype(kQUInt8),
+        weight.q_scale(),
+        weight_zp);
+    auto* qnnp_w_data = qnnp_weight.data_ptr<c10::quint8>();
+    for (int i = 0; i < weight_contig.numel(); ++i) {
+      qnnp_w_data[i] = static_cast<c10::quint8>(w_data[i] + 128);
+    }
     auto wt_ptr =
         guts::make_unique<PackedConvWeightsQnnp>(PackedConvWeightsQnnp{
             guts::make_unique<qnnpack::PrePackConvWeights>(
```

Comment:
can you just skip prepacking here all together? and assign nullptr in this member? You're going to recompute the whole thing on first invocation anyway

Reply:
I agree - however do you think if someone is reading the code it may throw them off seeing this as null?
Also in the future I think we can remove bias from the pre-packing so it may be okay to leave this here and remove just the bias field later. 
If you feel strongly about setting this to nullptr, I can do so :)

Reply:
It's just a noop code - why have it? I feel it throws off more to have some computation which is not used later

Reply:
Okay, changed it to nullptr in pre-pack :)